example/lib/main.dart

```diff
@@ -1,5 +1,6 @@
 import 'dart:async' show StreamSubscription;
 import 'dart:io';
+import 'package:pretty_json/pretty_json.dart';
```

Comment:
Have you tried using this https://api.dart.dev/stable/3.4.0/dart-convert/JsonEncoder/JsonEncoder.withIndent.html ? I prefer we stick with the stdlib.

Reply:
Updated